code/package.json

```diff
@@ -4,14 +4,18 @@
   "private": true,
   "dependencies": {
     "@babel/eslint-parser": "^7.18.9",
+    "@reduxjs/toolkit": "^1.9.0",
     "eslint": "^8.21.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "prop-types": "^15.8.1",
```

Comment:
What is this one for? We did not have it so we are curios about it.

Reply:
It was added in one of the tries to get our lotties to display 🙃 It was added as a dependency in the step by step guide from the team session, so we tried adding it here as well. Didn't help though.